assets/src/edit-story/components/panels/document/excerpt/excerpt.js

```diff
@@ -19,17 +19,24 @@
  */
 import { useCallback, useRef } from 'react';
 import { __ } from '@web-stories-wp/i18n';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
 import { useStory } from '../../../../app/story';
 import { Row, TextArea } from '../../../form';
 import { SimplePanel } from '../../panel';
-import Note from '../../shared/note';
 import { useFocusHighlight, states, styles } from '../../../../app/highlights';
+import { Text, THEME_CONSTANTS } from '../../../../../design-system';
 
-export const EXCERPT_MAX_LENGTH = 200;
+// Margin -4px is making up for extra margin added by rows.
+const StyledText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.tertiary};
+  margin-top: -4px;
```

Comment:
If we don't want this to be negative we could style the row here instead, but I think this is great too